deploy/charts/venafi-kubernetes-agent/templates/venafi-connection-crd.without-validations.yaml

```diff
@@ -1,10 +1,15 @@
----
+{{- if .Values.crds.enabled }}
+{{- if (or (semverCompare "<1.25" .Capabilities.KubeVersion.GitVersion) .Values.crds.forceRemoveValidationAnnotations) }}
```

Comment:
Thanks for fixing this. I didn't pay enough attention to these two CRD template files.